shared/app.js

```diff
@@ -92,10 +93,11 @@ module.exports = Backbone.Model.extend({
       this.templateAdapter = this.options.templateAdapterInstance;
     } else {
       var templateAdapterModule = attributes.templateAdapter || this.defaults.templateAdapter,
-      templateAdapterOptions = {entryPath: entryPath};
+        templateAdapterOptions = {entryPath: entryPath},
+        templateEngine = require(this.defaults.templateEngine);
```

Comment:
that makes sense to me, but while trying it I couldn't get it to work using simply `attributes`. Maybe because this is still happening in the constructor and the `defaults` are not set yet? Not sure..